wireless_modules/middle_module/test.py

```diff
@@ -0,0 +1,64 @@
+import paho.mqtt.client as mqtt
+import time
+import argparse
+
+
+parser = argparse.ArgumentParser(
+    description='MQTT wireless module test script with dummy payloads',
+    add_help=True)
+parser.add_argument(
+    '--module', action='store', type=int,
+    default=2, help="""The module number to determine the publish and subscription topics. 
+    If nothing is selected it will send on module number 2 topics.""")
+parser.add_argument(
+    '--host', action='store', type=str, default="5.196.95.208",
+    help="""Address of the MQTT broker. If nothing is selected it will
+    default to 5.196.95.208 - a test server by mosquito.""")
+
+
+if __name__ == "__main__":
+    client = mqtt.Client('my_id')
```

Comment:
If you leave out the client id, the library will choose a random client id, which is fine. By hardcoding the id we risk getting collisions if two people run the same script.